<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>passwordGenerator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/resources/icons/password-generator.ico</normaloff>:/icons/resources/icons/password-generator.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="locale">
   <locale language="Arabic" country="Libya"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="passwordGeneratorPushButton">
          <property name="styleSheet">
           <string notr="true">width:130px;</string>
          </property>
          <property name="text">
           <string>إنشاء كلمة مرور</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/images/resources/images/password-generator.png</normaloff>:/images/resources/images/password-generator.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="passwordLengthSpinBox"/>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="copyPasswordPushButton">
          <property name="styleSheet">
           <string notr="true">width:130px;</string>
          </property>
          <property name="text">
           <string>نسخ</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/images/resources/images/copy.png</normaloff>:/images/resources/images/copy.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="symbolsCheckBox">
          <property name="text">
           <string>رموز</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="numbersCheckBox">
          <property name="text">
           <string>أرقام</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="lowerCapitalscheckBox">
          <property name="text">
           <string>حروف صغيرة</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="upperCapitalsCheckBox">
          <property name="text">
           <string>حروف كبيرة</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
